### DISCUSSION TITLE
Extension activation failed while using Remote-SSH in VSC

### DISCUSSION THREAD

**Discussion by handflucht** [community/community#14175] [Copilot Conversations]
If I use GitHub Copilot as extension in Visual Studio Code in a local environment, it works fine. 
Now I want to use it with the extension "Remote - SSH" on a host, which hasn't any internet-connection. If I try doing so, I receive always the following error:

`[ERROR] [default] [2022-04-03T10:46:36.642Z] GitHub Copilot could not connect to server. Extension activation failed: "getaddrinfo ENOTFOUND api.github.com"
`

I think the extension tries to connect using a non-existing internet-connection of the remote-host instead via the client.  Is there a possiblity to configure/forward ports to **support GitHub Copilot also on a remote-Host which doesn't have any internet connection?**

Tested with the following extensions:
- GitHub.copilot (1.4.2022, 01:17:55)
- GitHub.copilot-nightly (3.4.2022, 03:39:48)

**Comment by amaruki** [comment]
have you try this?

https://github.com/github/feedback/discussions/7704#discussioncomment-1708369
